src/ChainSafe.GamingSDK.EVM/RPC/Contracts/Contract.cs

```diff
@@ -113,7 +113,9 @@ public async Task<object[]> Send(string method, object[] parameters = null, Tran
             if (txReq.GasPrice == null && txReq.MaxFeePerGas == null)
             {
                 var feeData = await provider.GetFeeData();
-                txReq.MaxFeePerGas = new HexBigInteger(feeData.MaxFeePerGas);
+
+                // txReq.MaxFeePerGas = new HexBigInteger(feeData.MaxFeePerGas);
```

Comment:
Is there a problem with `MaxFeePerGas`? Should we initialize GasPrice now instead?

Reply:
That was test to try to get `GasPrice`. Good point. I think it make sense to think about choosable strategy to tackle this.

Let me think how to properly refactor it.